docs/content/users/install/docker-installation.md

```diff
@@ -39,6 +39,12 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
 
     #### Migrating Projects from Docker Desktop to Colima
 
+    !!!tip "Docker Desktop may have left a bad config.json"
+        Remove the `credsStore` line in `~/.docker/config.json` if you have trouble.
```

Comment:
Qualify what kind of trouble makes this relevant so it’s less broad and open-ended:

```suggestion
        Remove the `credsStore` line in `~/.docker/config.json` if you have trouble running `ddev start` with a project you’ve migrated.
```